scipy/optimize/tests/test__differential_evolution.py

```diff
@@ -512,6 +516,8 @@ def test_deferred_updating(self):
         assert_(solver._mapwrapper._mapfunc is map)
         solver.solve()
 
+    @pytest.mark.skipif(IS_PYPY,
```

Comment:
if I'm reading the traceback correctly, we'll need an `import pytest` above too

Reply:
Yep, just pushed. And there was a failure in `test__dual_annealing`, possibly because `seed` was not set (and skipping this test might have had some effect on the randomization?). I've set it now to see if it fixes it, too.